src/containers/BellScheduleContainer.js

```diff
@@ -142,7 +142,9 @@ class BellScheduleContainer extends React.PureComponent<Props, State> {
       );
 
       const now = this.state.refreshed;
-
+      // now
+      // .hour(9)
+      // .minute(30)
```

Comment:
```suggestion
```
These comments don't add to the code.

Reply:
They allow for easy debugging, so I left them in for the review.

Reply:
I'm not an active maintainer of this app, so perhaps this is code standard here... but everywhere else I've worked, this sort of thing would be better suited for a comment. It's easier to forget about like this 😄 .